src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -115,7 +115,7 @@ const SubmissionDetails = _.flow(
 
   // Note: This 'deleteLimitYears' value should reflect the current 'deletion-delay' value configured for PROD in firecloud-develop's
```

Comment:
Nit: This comment appears to be out-of-date with respect to the const name. Not part of the change in this PR, but probably worth fixing.